lib/core/src/display/scene/htmlrenderer.rs

```diff
@@ -0,0 +1,104 @@
+use super::{Camera, HTMLScene};
+
+fn eps(value: f32) -> f32 {
+    if value.abs() < 1e-10 {
+        0.0
+    } else {
+        value
+    }
+}
+
+pub struct HTMLRenderer {}
+
+impl HTMLRenderer {
+    pub fn new() -> Self {
+        Self {}
+    }
```

Comment:
Each struct which implements `new` without args should have `Default` instance. Plus, this is a special situation when the function is so short, we definitely can make it one line long

Reply:
I just derived it from Default :+1: